anchor/common/ssv_types/src/share.rs

```diff
@@ -0,0 +1,8 @@
+use types::PublicKey;
+
+/// One of N shares of a split validator key.
+#[derive(Debug, Clone)]
+pub struct Share {
```

Comment:
Should we add the (encrypted) private key here?

Reply:
I had it in but removed it right before the PR as I convinced myself it wasn't needed in the Share for some reason. Thinking on it now it should probably be here. I'll add it back in thanks!